docs-sources/includes/_Archivist.md

```diff
@@ -496,3 +524,74 @@ to only data which matches the userId.
 
 We then use the acl function to only allow users and tests access to
 the `get` API, but full access to CMS users and administrators.
+
+## Migrations
+
+MAGE supports database migration scripts similar to [Ruby on Rails 2.1](http://api.rubyonrails.org/classes/ActiveRecord/Migration.html)
+which are aligned with your `package.json` version and exposed via the `archivist:migrate`
```

Comment:
@ronkorving is this correct? I am pretty sure vault versions have nothing to do with the version in `package.json`